src/getExtendedPublicKey.c

```diff
@@ -74,18 +74,18 @@ void io_exchange_address()
 {
 	uint32_t tx = 0;
 
-	G_io_apdu_buffer[tx++] = 32;
+	G_io_apdu_buffer[tx++] = PUBLIC_KEY_LENGTH;
 
-	uint8_t rawPublicKey[32];
+	uint8_t rawPublicKey[PUBLIC_KEY_LENGTH];
 	extractRawPublicKey(&data.publicKey, rawPublicKey);
-	os_memmove(G_io_apdu_buffer + tx, rawPublicKey, 32);
-	os_memset(rawPublicKey, 0, 32);
+	os_memmove(G_io_apdu_buffer + tx, rawPublicKey, PUBLIC_KEY_LENGTH);
+	os_memset(rawPublicKey, 0, PUBLIC_KEY_LENGTH);
```

Comment:
Unlike private keys I don't think there is a particular need to clear public key here...